apps/verticalhome/test/marionette/grid_layout_test.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+var assert = require('assert');
+
+marionette('Vertical - grid', function() {
+
+  var client = marionette.client(require(__dirname + '/client_options.js'));
+  var actions, home, system;
+
+  setup(function() {
+    actions = client.loader.getActions();
+    home = client.loader.getAppClass('verticalhome');
+    system = client.loader.getAppClass('system');
+    system.waitForStartup();
+    home.waitForLaunch();
+  });
+
+  test('apps grid is LTR in LTR language', function() {
+    var communicationManifestUrl =
+        'app://communications.gaiamobile.org/manifest.webapp';
+    var phoneIcon = home.getIcon(communicationManifestUrl, 'dialer').location();
+    var contactsIcon = home.getIcon(communicationManifestUrl, 'contacts')
+      .location();
+    var messageIcon = home.getIcon('app://sms.gaiamobile.org/manifest.webapp')
+      .location();
```

Comment:
This relies on us installing these apps and never changing their order. It would make more sense to just get the first two app icons and test that the first is aligned to the left, and that the second is to the right of the first.